plugins/modules/ios_ospfv2.py

```diff
@@ -30,23 +30,17 @@
 
 __metaclass__ = type
 
-ANSIBLE_METADATA = {
-    "metadata_version": "1.1",
-    "status": ["preview"],
-    "supported_by": "network",
-}
 
 DOCUMENTATION = """
----
 module: ios_ospfv2
-version_added: 2.9
-short_description: OSPF_v2 resource module.
-description: This module configures and manages the Open Shortest Path First (OSPF) version 2 on IOS platforms.
+short_description: OSPFv2 resource module
+description: This module configures and manages the Open Shortest Path First (OSPF)
+  version 2 on IOS platforms.
+version_added: 1.0.0
 author: Sumit Jaiswal (@justjais)
 notes:
-  - Tested against Cisco IOSv Version 15.2 on VIRL
-  - This module works with connection C(network_cli).
-    See L(IOS Platform Options,../network/user_guide/platform_ios.html).
+- Tested against Cisco IOSv Version 15.2 on VIRL
+- This module works with connection C(network_cli). See L(IOS Platform Options,../network/user_guide/platform_ios.html).
```

Comment:
Link to platform guide is no longer valid in collections.

Reply:
ah yes, removal should've been made part of update script xD